### PR TITLE
add attachments apis

### PR DESCRIPTION
**Pull request by Hyjaz**
## Request for Change

[Description and reason for the change](https://zendesk.atlassian.net/jira/software/projects/SCC/boards/1780?selectedIssue=SCC-404)

Added 2 more apis to the smooch postman collection under attachments API folder [@here](https://github.com/zendesk/sunshine-conversations-postman-collection-public/pull/19/files#diff-4b081629b5ae173add1b20828fb6033bR2770-R3172)

- Generate Media Token
- Get Cookie

Priority : `low` 
Impact : `low` 
Risk : `low` 
Classification : `preauthorized` 
Security impact: None 

Automated tests: N/A
Manual tests: N/A 

Expected deployment date/time : DATE_TIME 
Expected downtime duration : 0 minutes
Rollout plan & rollback instructions: Revert github files and push to master

### CODE REVIEW

File: Smooch.postman_collection.json [hunk 1]
```diff
@@ -1328,7 +1328,7 @@
 					},
 					"response": [
 						{
-							"name": "Delete App User Example",
+							"name": "Delete App User Profile Example",
```
Comment (maxlacoste92) [inline Smooch.postman_collection.json:1331]: Why is this updated?

Comment (maxlacoste92) [reply Smooch.postman_collection.json:1331]: I just noticed you re-ordered the apis. Why?

**Review by maxlacoste92 — APPROVED** [review]

File: Smooch.postman_collection.json (hunk 1 above)
Comment (Hyjaz) [reply Smooch.postman_collection.json:1331]: I had only added the generate media token and the get token api. The rest were changed by themselves. There is a thread on [#sunco-eng](https://zendesk.slack.com/archives/CJM6HKQUA/p1595363412301000) where I asked the exact same question.

Comment (DataWearsAHood) [reply Smooch.postman_collection.json:1331]: Postman has an annoying tendency to move entry-order around when editing/rebundling a collection; sorry bro.

File: Smooch.postman_collection.json [hunk 2]
```diff
@@ -8926,10 +9001,10 @@
...
+									"body": "{\"message\":{\"text\":\"1. Scandinave Getaway\\nEnjoy massage treatments situated in a serene natural forest with an on-site bistro serving healthy local fare\\n\",\"items\":[{\"title\":\"Scandinave Getaway\",\"description\":\"Enjoy massage treatments situated in a serene natural forest with an on-site bistro serving healthy local fare\",\"mediaUrl\":\"https://www.scandinave.com/app/uploads/sites/5/2017/01/2015-upper-hot-bath-hp-820x779.jpg\",\"mediaType\":\"image/jpeg\",\"_id\":\"5b2cf9653e3adc00228ba421\",\"actions\":[{\"_id\":\"5b2cf9653e3adc00228ba422\",\"uri\":\"\",\"type\":\"share\"}]}],\"type\":\"carousel\",\"role\":\"appMaker\",\"received\":1529674085.179,\"authorId\":\"00uejytoekz520ZEG0h7\",\"avatarUrl\":\"https://media.smooch.io/5ac3da9539fe6c0022447259/icons/appicon.jpg\",\"name\":\"Galifrey\",\"_id\":\"5b2cf9653e3adc00228ba420\",\"source\":{\"type\":\"api\"}},\"conversation\":{\"unreadCount\":2,\"_id\":\"884ac26bfa519a0bb4cb0162\",\"appMakerLastRead\":1529674085.179,\"appUserLastRead\":1529672980.196}}"
 								},
 								{
-									"name": "Action:  share Example",
```
Comment (DataWearsAHood) [inline Smooch.postman_collection.json:8932]: any [relatively simple] way to flag this extra space for the next person to update/export the collection? 🤔

**Review by DataWearsAHood — APPROVED** [review]
Passes visual check against docs (PR) 👍

**Comment by Hyjaz** [comment]
opened in different pr https://github.com/zendesk/sunshine-conversations-postman-collection-public/pull/22

**Pull request closed without merging**